src/lib/wdi5-bridge.ts

```diff
@@ -92,8 +92,8 @@ function initBrowser(browserInstance: WebdriverIO.Browser) {
 
     _addWdi5Commands(browserInstance)
 
-    if (!(browserInstance as any).fe) {
-        ;(browserInstance as any).fe = WDI5FE
+    if (!(browser as any).fe) {
```

Comment:
hm, `browserInstance` is explicitly passed in here → why is it not supposed to be used? IMO the `browserInstance` helps retain scope within the fn by explicitly not referencing the global `browser`

Reply:
Then we would have to initialize the facade as often as we have defined browsers in the config. That would look like something like this: 

```javascript
fioriElementsFacadeOne = await one.fe.initialize{...}

fioriElementsFacadeTwo = await two.fe.initialize{...}
```
We can do that of course, I just don't now if this is the best way